client/my-sites/stats/activity-log-banner/success-banner.jsx

```diff
@@ -71,16 +71,22 @@ class SuccessBanner extends PureComponent {
 	render() {
 		const { applySiteOffset, moment, siteUrl, timestamp, translate, backupUrl } = this.props;
 		const date = applySiteOffset( moment.utc( ms( timestamp ) ) ).format( 'LLLL' );
+		const params = backupUrl
+			? {
+					title: translate( 'Your backup is now available for download' ),
+					icon: 'cloud-download',
+				}
+			: {
+					title: translate( 'Your site has been successfully restored' ),
+					icon: 'history',
+				};
+
 		return (
 			<ActivityLogBanner
 				isDismissable
 				onDismissClick={ this.handleDismiss }
 				status="success"
-				title={
-					backupUrl
-						? translate( 'Your backup has been successfully created' )
-						: translate( 'Your site has been successfully restored' )
-				}
+				{ ...params }
```

Comment:
I'd be inclined here to split up `title` and `icon` so that the props being passed down are super explicit. Since they are located right above it's not the biggest deal in the world, but definitely in general it's confusing to find these objects being passed directly as props because it's not clear what is in them and what isn't.

One idea is to split them in their declarations…

```js
const title = backupUrl ? translate( … ) : translate( … )
const icon = backupUrl ? 'cloud-download' : 'history'

…

title={ title }
icon={ icon }
```

another option is to leave the declaration as-is but make the passing explicit

```js
status="success"
title={ params.title }
icon={ params.icon }
```

a third option is to just destructure here instead of using `params`

```js
const { icon, title } = backupUrl
    ? { title: translate( 'Your …' ), icon: 'cloud-download' }
    : { title: translate( 'Your …' ), icon: 'history' }
```

Reply:
Ok I'll make the passing explicit in the attributes